Create SECURITY.md (#115)
sora-xor/sora-docs: SECURITY.md

# Security Policy

## Reporting a Vulnerability
If you think you have discovered a security issue in the SORA docs, please let us know. 

> We take all security bugs seriously and if confirmed upon investigation we will patch it as soon as possible and release a public security bulletin discussing the impact and crediting the discoverer. 

Reviewer: ```suggestion
We take all security bugs seriously. If confirmed upon investigation, the issue will be addressed as soon as possible. We will release a public security bulletin discussing the impact and crediting the discoverer. 
```

> Reporting a security weakness is easy - just email us a description of the flaw and any related information (e.g. reproduction steps, version) to sora@soramitsu.co.jp

Reviewer: ```suggestion
To report a security issue, send the description of the flaw and any related information (e.g. reproduction steps, version) to [sora@soramitsu.co.jp](mailto:sora@soramitsu.co.jp).
```
